webdriver/tests/support/fixtures.py

```diff
@@ -227,8 +227,8 @@ async def bidi_session(capabilities, configuration):
 
     yield _current_session.bidi_session
 
+    await cleanup_bidi_session(_current_session)
     await _current_session.bidi_session.end()
-    cleanup_session(_current_session)
```

Comment:
FYI we should make sure that no interop test between classic and BiDi will leave traces like open alerts around, which would break the teardown now at least until we can handle those extra features in BiDi.

Reply:
Ack. Experimenting with calling `cleanup_bidi_session` and then `cleanup_session`.